src/main/java/com/spotify/reaper/storage/CassandraStorage.java

```diff
@@ -184,19 +194,15 @@ public RepairRun addRepairRun(Builder repairRun, Collection<RepairSegment.Builde
             newRepairRun.getCause(),
             newRepairRun.getOwner(),
             newRepairRun.getRunState().toString(),
-            newRepairRun.getCreationTime()==null?null:newRepairRun.getCreationTime(),
-            newRepairRun.getStartTime()==null?null:newRepairRun.getStartTime(),
-            newRepairRun.getEndTime()==null?null:newRepairRun.getEndTime(),
-            newRepairRun.getPauseTime()==null?null:newRepairRun.getPauseTime(),
+            newRepairRun.getCreationTime(),
+            newRepairRun.getStartTime(),
+            newRepairRun.getEndTime(),
+            newRepairRun.getPauseTime(),
```

Comment:
I'm pretty sure I did this because we had some problems with null values... 
I'll have to test this again before merging.